Hughsie/elantp i2c

hughsie opened the pull request:
Type of pull request:

- [ ] New plugin (Please include [new plugin checklist](https://github.com/fwupd/fwupd/wiki/New-plugin-checklist))
- [ ] Code fix
- [ ] Feature
- [ ] Documentation

hughsie commented:
@josh-chen-elan @jinglewu

josh-chen-elan reviewed (commented):
This quirk should be kept, ACPI\ELAN0000 does exist on most Chromebooks: [https://chromium.googlesource.com/chromiumos/third_party/kernel/+/refs/heads/chromeos-6.6/include/linux/input/elan- i2c-ids.h#23](https://chromium.googlesource.com/chromiumos/third_party/kernel/+/refs/heads/chromeos-6.6/include/linux/input/elan- i2c-ids.h#23)

hughsie commented:
> This quirk should be kept, ACPI\ELAN0000 does exist on most Chromebooks

Oopa, my mistake -- thanks. I've reverted this patch and added a new patch that appends `I2C\NAME_ELAN0670:00`
